### PR TITLE
[tdlibjson_wrapper][models] add VoiceNote and Audio handling

### PR DESCRIPTION
**Pull request by Nokius**
if I'm not to sleepy by now this should help to handle the Audio and VoiceNote in depecher.
The QML PART is **NOT** included here.

I checked this two doc
* [audio](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1animation.html)
* [voicenote](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1voice_note.html)
   * I didn't got the waveform_ so I didn't added it for now, it looks to me it's for the visual feature as in tdesktop

### CODE REVIEW

File: tdlibjson_wrapper/tdlibQt/models/MessagingModel.hpp [hunk 1]
```diff
@@ -70,6 +70,8 @@ class MessagingModel : public QAbstractListModel
         FILE_CAPTION,
         PHOTO_ASPECT,
         DOCUMENT_NAME,
+	VOICE_DURATION,
```
Comment (frzifus) [inline tdlibjson_wrapper/tdlibQt/models/MessagingModel.hpp]:
```suggestion
        VOICE_DURATION
```
Maybe we should just indent this line properly. :b

Comment (Nokius) [reply tdlibjson_wrapper/tdlibQt/models/MessagingModel.hpp]: Thanks for the Hint I added it including the comma at the end =)

Comment (Nokius) [reply tdlibjson_wrapper/tdlibQt/models/MessagingModel.hpp]: and never ever use tabs :D

**Comment by frzifus** [comment]
i'm not quite sure but i think we need to parse the audio messages [here](https://github.com/blacksailer/depecher/blob/master/tdlibjson_wrapper/tdlibQt/ParseObject.hpp) and map them to the model.

See [L724](https://github.com/blacksailer/depecher/blob/master/tdlibjson_wrapper/tdlibQt/ParseObject.cpp#L724) [L915](https://github.com/blacksailer/depecher/blob/master/tdlibjson_wrapper/tdlibQt/ParseObject.cpp#L915)

**Pull request merged**